src/components/util/Bubble.tsx

```diff
@@ -101,6 +117,31 @@ export default function Bubble(props: Props) {
 		setIsEditing(false);
 	};
 
+	const bubbleSwitch = (e: { keyCode: any }, selectNext?: () => void, selectPrev?: () => void) => {
```

Comment:
bubbleSwitch will receive KeyboardEvent as parameter,
there is no need to accept functions as parameters.
we can just call them inside this function;

we need to check if event.target is input to access e.target.selectionStart
```
if(e.target instanceof HTMLInputElement) {
  const selectionStart = e.target.selectionStart
}
```